prow/config.yaml

```diff
@@ -11244,7 +11244,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- interval: 1h
+- cron: '0 0 25 4 *' # Run at 00:00 UTC (02:00 CEST) on 25th April
```

Comment:
Can you change it to 00:01? (just for my satisfaction that it's actually counted as part of 25th april:)